portal-api/package.json

```diff
@@ -35,7 +35,7 @@
   "dependencies": {
     "@azure/storage-file-share": "12.11.0",
     "@graphql-tools/schema": "^8.5.0",
-    "apollo-server-express": "2.18.2",
+    "apollo-server-express": "3.11.1",
```

Comment:
there are breaking changes in v3. we either update to the latest v2. or we need to fix the breaking changes

Reply:
Downgraded to 2.26.1